app.py

```
@@ -46,7 +46,31 @@ def ping():
         
 @app.route("/comports")
 def comports():
-    return serial_connection.available_comports()
+    try:
+        with serial_lock():
+            comports = serial_connection.available_comports()
+
+        if comports:
+            print(str(comports))
+            return jsonify(comports), 200
+        else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)